RobotConstantsBase.java

```diff
@@ -40,6 +40,11 @@ public  static class SwerveDriveBase {
         public static final double kMaxDriveCurrent = 70.0;//Defaults
         public static final double kMaxSteerCurrent = 30.0;//Defaults
 
+        public static final double kCurrentSampleTimeRange = 5;
+        public static final int kCurrentSampleSize = (int) (kCurrentSampleTimeRange * 50); // mustangPeriodic runs 50hz
+        public static final double kTotalDriveCurrentThreshold = 120; // 30 per motor
+        public static final double kDriveCurrentLimit = 25; // single motor 
```

Comment:
Maybe name this something so it's clearer that this is a lowered limit for the case that total current is getting too high